frontend/src/layout/navigation-3000/sidepanel/panels/activity/notificationsLogic.tsx

```diff
@@ -0,0 +1,245 @@
+import { actions, events, kea, listeners, path, reducers, selectors } from 'kea'
```

Comment:
Is this file meant to be in this PR? I don't see it referenced from any of the data warehouse stuff.

Reply:
bad merge!